src/dotnet/APIView/APIViewWeb/Repositories/PullRequestManager.cs

```diff
@@ -99,7 +99,8 @@ IConfiguration configuration
                 if (pullRequestModel == null || pullRequestModel.Commits.Any(c=> c== commitSha))
                 {
                     // PR commit is already processed. No need to reprocess it again.
-                    return "";
+                    return pullRequestModel != null && !string.IsNullOrEmpty(pullRequestModel.ReviewId)?

```

Comment:
Do you know in which cases `pullRequestModel.ReviewId` is null or empty? like what does that mean and why we need to validate it?

Reply:
Ideally,  `pullRequestModel.ReviewId` shouldn't be null. But I think it's better to add this check to avoid there is a wrong data stored in database, whose `pullRequestModel.ReviewId` is null.